# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tar/compar_does_not_need_fd.h.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---
--- ./src/compare.c.orig	2004-06-24 13:49:08.444825752 +0200
+++ ./src/compare.c	2004-06-24 13:47:15.575984424 +0200
@@ -31,9 +31,11 @@
   };
 #endif
 
+#if 0
 #if HAVE_LINUX_FD_H
 # include <linux/fd.h>
 #endif
+#endif
 
 #include <quotearg.h>
 
